appconfiguration/spring-cloud-azure-feature-management-web/spring-cloud-azure-targeting-filter-web-sample/src/main/java/com/example/demo/controllers/HelloController.java

```diff
@@ -0,0 +1,26 @@
+package com.example.demo.controllers;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+
+import com.azure.spring.cloud.feature.management.FeatureManager;
+import org.springframework.web.bind.annotation.GetMapping;
+
+
+@Controller
+@ConfigurationProperties("controller")
```

Comment:
You shouldn't need this line it isn't an `@ConfigurationProperties`

Reply:
Updated